Recruit-Training/Embedded-Recruit-Training/Core/Src/main.c

```diff
@@ -293,6 +323,31 @@ void HAL_CAN_RxFifo0MsgPendingCallback(CAN_HandleTypeDef* hcan)
     }
 
     //TODO: Match StdId of header and data length content for green and blue messages and check data for set bit of toggling green and blue led
+    if (hdr.DLC == 1){
+        if(hdr.StdId == 0xAAA)
```

Comment:
Use  what you've already defined (`BLUE_MESSAGE_STDID`) instead of `0xAAA`. This is called a magic number and it's bad practice since reader might not know what you mean by `0xAAA`.

https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad